src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -82,6 +87,12 @@ def ramp_fit(model, buffsize, save_opt, readnoise_2d, gain_2d,
         Object containing optional GLS-specific ramp fitting data for the
         exposure
     """
+    constants.dqflags["DO_NOT_USE"] = dqflags["DO_NOT_USE"]
+    constants.dqflags["SATURATED"] = dqflags["SATURATED"]
+    constants.dqflags["JUMP_DET"] = dqflags["JUMP_DET"]
+    constants.dqflags["NO_GAIN_VALUE"] = dqflags["NO_GAIN_VALUE"]
+    constants.dqflags["UNRELIABLE_SLOPE"] = dqflags["UNRELIABLE_SLOPE"]
```

Comment:
You could do `constants.dqflags.update(dqflags)` to update the entire dict at once.
We probably need to make sure all flags have a valid value
```
if None in constants.dqflags.values():
   raise ValueError
```

Reply:
I created an update function and a `NoneType` check.